File: 02-FEA-1D-G/Matlab-Code/PrintSolution.m
function PrintSolution(UF,PP,UUR,PUR)

fout = fopen('UF_Vector.dat','w');
fprintf(fout, '%12.8f \n', UF);
fclose(fout); 

fout = fopen('PP_Vector.dat','w');
fprintf(fout, '%12.8f \n', PP);
fclose(fout); 

fout = fopen('UUR_Vector.dat','w');
fprintf(fout, '%12.8f \n', UUR);
fclose(fout); 

fout = fopen('PUR_Vector.dat','w');
fprintf(fout, '%12.8f \n', PUR);
fclose(fout); 

fprintf('NODE DISPLACEMENT\n')
fprintf( '%12.8f \n', UUR);
fprintf('NODE FORCE\n')
fprintf( '%12.8f \n', PUR);


end

File: 06-FEA-3D-Elasticity/C-Code/PrintSolution.cpp
#include <MainFile.h>
using namespace std;

void PrintSolution(arma::colvec& Uf, arma::colvec& Pp,  arma::mat& Uur,   arma::mat& Pur)
{

  ofstream fout;

  fout.open ("UF_Vector.dat");
  fout << Uf;
  fout.close();
 
  fout.open ("PP_Vector.dat");
  fout << Pp;
  fout.close();

  fout.open ("UUR_Vector.dat");
  fout << Uur;
  fout.close();

  fout.open ("PUR_Vector.dat");
  fout << Pur;
  fout.close();


  cout<<"NODE DISPLACEMENT\n";
  cout<<Uur<<endl;
  cout<<"NODE FORCE\n";
  cout<<Pur<<endl;

}
